test/indexer/test_indexer_controller.py

```diff
@@ -89,12 +90,41 @@ def test_invalid_notification(self):
         self.assertRaises(AssertionError, self.controller.contribute, event)
 
     def test_remote_reindex(self):
-        event = [self._mock_sqs_record(action='reindex', prefix='ff')]
-        with mock.patch.object(target=AzulClient,
-                               attribute=AzulClient.remote_reindex_partition.__name__) as mock_method:
-            mock_method.return_value = True
-            self.controller.contribute(event)
-            mock_method.assert_called_once_with(dict(action='reindex', prefix='ff'))
+        mock_ppl = mock.patch.dict(os.environ, AZUL_PARTITION_PREFIX_LENGTH='0')
```

Comment:
patch.dict accepts multiple kwargs

Reply:
Even if it didn't you don't need to assign individual variables. A single tuple-typed variable would work. or you could just inline all three and wrap the `with` statement.